src/v3/src/components/Form/Form.tsx

```diff
@@ -78,7 +78,7 @@ const Form: FunctionComponent<{
       onSubmit={handleSubmit}
       className="o-form" // FIXME update page objects using .o-form selectors
       data-se="form"
-      style={{ maxWidth: '100%' }}
+      style={{ maxWidth: '100%', wordBreak: 'break-word' }}
```

Comment:
Can you look into these MDN docs? It looks like `word-break: break-word` is deprecated but there's another relevant property called `overflow-wrap`:
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
🤦 I did not know this was deprecated (TIL), looking at the docs overflow-wrap is definitely the property to use, I am now going to question every prop that I have used before to make sure it is still viable 😅 

TY!